ui/src/timeMachine/constants/queryBuilder.ts

```diff
@@ -6,77 +6,103 @@ export interface QueryFn {
   flux: (period?: string) => string
 }
 
+export const genFlux = (func: string, period?: string) => {
+  if (period === 'none') {
```

Comment:
should this be checking against the AGG_WINDOW_NONE rather than the string `none`?